.github/workflows/build-main-branches.yml

```diff
@@ -5,8 +5,7 @@ on:
     push:
         branches:
             - main
-            - 2.x
```

Comment:
We may need to communicate that 2.x is now abandoned.

Reply:
I am going to make an announcement in Discussions.